var/spack/repos/builtin/packages/py-nilearn/package.py

```diff
@@ -53,29 +56,10 @@ class PyNilearn(PythonPackage):
     depends_on("py-matplotlib@1.1.1:3.3", when="@:0.6 +plotting", type=("build", "run"))
```

Comment:
Dependencies need to be updated:
```
install_requires =
        joblib>=0.15
        lxml
        nibabel>=3.0.0
        numpy>=1.18
        pandas>=1.0
        requests>=2
        scikit-learn>=0.22
        scipy>=1.5

[options.extras_require]
plotting =
        matplotlib>=3.0
```

Reply:
Sorry, I seems that I only checked the dependencies for version 0.9.0 and missed the once for 0.9.2. This is fixed now.